plugins/cassandra/cassandrawu.cpp

```diff
@@ -2462,7 +2462,7 @@ class CCassandraWorkUnit : public CPersistedWorkUnit
             assertex(row);
             StringBuffer b;
             getCassString(b, cass_row_get_column(row, 0));
-            graphName.set(b);
+            graphName.set(std::move(b));
```

Comment:
Is it safe to use b after the move? If not, is there something we can do to prevent misuse?